src/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -61,7 +63,7 @@ public function __call($method, $args)
     {
         if ('persistAs' === substr($method, 0, 9)) {
             if (!isset($args[0])) {
-                throw new \Gedmo\Exception\InvalidArgumentException('Node to persist must be available as first argument');
+                throw new InvalidArgumentException('Node to persist must be available as first argument');
```

Comment:
```suggestion
                throw new InvalidArgumentException('Node to persist must be available as first argument.');
```

Reply:
Doesn't this qualify as a BC change, in the unlikely scenario someone is checking for the exception message text?

Reply:
I honestly hope not. But if you think it may be the case, please dismiss my suggestion.